.travis.yml

```diff
@@ -1,5 +1,11 @@
-language: jruby
+language: ruby
 rvm:
-  - 1.7.16.1
+rvm:
```

Comment:
Is there a reason that this isn't just 2.1.5, or maybe 1.9.3, 2.1.5, and perhaps jruby-1.7.16.1?